src/astarte-client/models/Block/CustomBlock.ts

```diff
@@ -0,0 +1,40 @@
+/*
+   This file is part of Astarte.
+
+   Copyright 2020 Ispirata Srl
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+import { BlockType, CustomBlockDTO } from '../../types';
+
+export class CustomBlock {
+  name: string;
+
+  source: string;
+
+  type: BlockType;
+
+  schema: Record<string, unknown>;
+
+  constructor(block: CustomBlockDTO) {
+    this.name = block.name;
+    this.source = block.source;
+    this.type = block.type;
+    this.schema = block.schema;
+  }
+}
+
+export const toCustomBlock = (dto: CustomBlockDTO) => {
```

Comment:
**to** functions are usually class functions, like `.toString` so it makes sense to read
`number.toString()`. For static functions (or generator in general) it's common to use **from**
`CustomBlock.fromJsonString(string)`